code/_macros.dm

```diff
@@ -75,6 +75,11 @@
 
 #define isCrowbar(A) istype(A, /obj/item/weapon/tool/crowbar)
 
+// Overlays
+// (placeholders for if/when TG overlays system is ported)
+#define cut_overlays(...)			overlays.Cut()
```

Comment:
This seems like a bad use of defines, I think a stub proc would work better here. Something like:
```
/atom/proc/cut_overlays(x)
    overlays.Cut()
```

Reply:
It's a stub for if/when tgstation overlay manager gets ported. Same as `to_chat`.